### PR TITLE
Add equalPassword rule

### PR DESCRIPTION
**Pull request by arochedy**
Add a new rule to check confirm password form.

### CODE REVIEW

File: defaultRules.js [hunk 1]
```diff
@@ -27,7 +27,10 @@ const defaultRules = {
       return false;
     }
     return true;
-  }
+  },
+  equalPassword(dataToCompare, value) {
+    return dataToCompare == value;
```
Comment (perscrew) [inline defaultRules.js:32]: would be better no ?

```suggestion
    return dataToCompare === value;
```

**Review by perscrew — CHANGES_REQUESTED** [review]
1 small comment

File: defaultRules.js (hunk 1 above)
Comment (arochedy) [reply defaultRules.js]: Yes I would be better right. I Commited your comment

**Review by perscrew — APPROVED** [review]
👍

**Pull request merged**